### PR TITLE
Food places data

### LINKED ISSUE #53: Dining options data class skeleton
To be used for /dine and /dineinfo

### PR DESCRIPTION
**Pull request by wamikamalik**
Note: Some faculties/places have been combined due to proximity or shared food places,\.
Fix #53

### CODE REVIEW

File: src/main/java/seedu/duke/FoodPlace.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    private ArrayList<String> faculty = new ArrayList<>();
+    private ArrayList<DiningOptions> info = new ArrayList<>();
+
+    public FoodPlace(String[] faculty, String[][] data) {
+        this.faculty.addAll(Arrays.asList(faculty));
+        for (int i = 0; i < data.length; i++) {
+            info.add(new DiningOptions(data[i]));
+        }
+    }
+
```
Comment (EthanWong2212) [inline src/main/java/seedu/duke/FoodPlace.java:17]: Perhaps we should add getters for faculty and info here too. Otherwise we can go ahead and merge

**Review by EthanWong2212 — APPROVED** [review]

File: src/main/java/seedu/duke/FoodPlace.java (hunk 1 above)
Comment (wamikamalik) [reply src/main/java/seedu/duke/FoodPlace.java:17]: Sure! I was wondering if those should be added if and when they are being used...

Comment (EthanWong2212) [reply src/main/java/seedu/duke/FoodPlace.java:17]: Sure, we can add it when implementing the other features

File: src/main/java/seedu/duke/FoodPlacesData.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            {"Reedz Cafe", "The Thinking Corner @ BTC Li Ka Shing Building", "Mon-Fri, 8.00am-7.00pm"}
+    };
+
+    private static ArrayList<FoodPlace> diningOptionsInNus = new ArrayList<>();
+
+    public FoodPlacesData() {
+        diningOptionsInNus.add(new FoodPlace(new String[]{"Science", "University Hall", "Medicine"},
+                foodPlacesScience));
+        diningOptionsInNus.add(new FoodPlace(new String[]{"PGP"}, foodPlacesPgp));
+        diningOptionsInNus.add(new FoodPlace(new String[]{"School of Business", "School of computing",
```
Comment (mrwsy1) [inline src/main/java/seedu/duke/FoodPlacesData.java:102]:
```suggestion
        diningOptionsInNus.add(new FoodPlace(new String[]{"Business", "School of computing",
```

**Pull request merged**